apps/server/src/modules/tldraw/service/tldraw.ws.service.ts

```diff
@@ -133,45 +127,40 @@ export class TldrawWsService {
 	}
 
 	public messageHandler(conn: WebSocket, doc: WsSharedDocDo, message: Uint8Array): void {
-		try {
-			const encoder = encoding.createEncoder();
-			const decoder = decoding.createDecoder(message);
-			const messageType = decoding.readVarUint(decoder);
-			switch (messageType) {
-				case WSMessageType.SYNC:
-					encoding.writeVarUint(encoder, WSMessageType.SYNC);
-					readSyncMessage(decoder, encoder, doc, conn);
-
-					// If the `encoder` only contains the type of reply message and no
-					// message, there is no need to send the message. When `encoder` only
-					// contains the type of reply, its length is 1.
-					if (encoding.length(encoder) > 1) {
-						this.send(doc, conn, encoding.toUint8Array(encoder));
-					}
-					break;
-				case WSMessageType.AWARENESS: {
-					const update = decoding.readVarUint8Array(decoder);
-					this.publishUpdateToRedis(doc, update, 'awareness');
-					break;
+		const encoder = encoding.createEncoder();
+		const decoder = decoding.createDecoder(message);
+		const messageType = decoding.readVarUint(decoder);
+		switch (messageType) {
+			case WSMessageType.SYNC:
+				encoding.writeVarUint(encoder, WSMessageType.SYNC);
+				readSyncMessage(decoder, encoder, doc, conn);
+
+				// If the `encoder` only contains the type of reply message and no
+				// message, there is no need to send the message. When `encoder` only
+				// contains the type of reply, its length is 1.
+				if (encoding.length(encoder) > 1) {
+					this.send(doc, conn, encoding.toUint8Array(encoder));
 				}
-				default:
-					break;
+				break;
+			case WSMessageType.AWARENESS: {
+				const update = decoding.readVarUint8Array(decoder);
```

Comment:
Please move this logic to private method. Switch/case schould call a methode

Reply:
good idea, done